tests/examples/source/devfiles/nodejs/devfile-with-link.yaml

```diff
@@ -43,23 +43,57 @@ components:
   name: runtime
 - kubernetes:
     inlined: |
-      apiVersion: etcd.database.coreos.com/v1beta2
-      kind: EtcdCluster
+      apiVersion: redis.redis.opstreelabs.in/v1beta1
+      kind: RedisCluster
       metadata:
-        annotations:
-          etcd.database.coreos.com/scope: clusterwide
-        name: myetcd
+        name: myredis
       spec:
-        size: 1
-        version: 3.2.13
-  name: myetcd
+        clusterSize: 3
```

Comment:
Can we do with `clusterSize: 1`? I'm asking this because it would help us spin up operands with less resource requirements and help speed up tests. #4869 is along the same lines.

Reply:
`Redis` CRD does not have `clusterSize:`, so I have removed it completely 🙂